docs/api/covidcast-signals/doctor-visits.md

```diff
@@ -33,15 +33,20 @@ percentage of COVID-related doctor's visits in a given location, on a given day.
 Note that because doctor's visits may be reported to the health system partners
 several days after they occur, these signals are typically available with
 several days of lag. This means that estimates for a specific day are only
-available several days later.
+available several days later. 
 
 The amount of lag in reporting can vary, and not all visits are reported with
 the same lag. After we first report estimates for a specific date, further data
 may arrive about outpatients visits on that date. When this occurs, we issue new
-estimates for those dates. This means that a reported estimate for, say, June
+estimates for those dates to backfill any missing data. This means that a reported estimate for, say, June
 10th may first be available in the API on June 14th and subsequently revised on
 June 16th.
 
+ As insurance claims are available at a significant and variable latency, the signal experiences heavy backfill with data delayed for a couple of weeks. 
+We expect estimates available for the most recent 7-14 days to change substantially in later data revisions. Estimates for dates more than 28 days in the past are expected to remain fairly static, as most major revisions have already occurred.
```

Comment:
I'm not sure the same criteria for determining the "fairly static" threshold are the same for doctor-visits and chng. 94% of the chng data is in by 60 days, but the change in signal values might still be negligible before that since we are missing data from both the numerator and denominator.

Reply:
that's fine -- do you have alternate figures for dv and chng we can use to give people a more consistent idea of what to expect?

Reply:
@rumackaaron ping